Baekjoon/1546.py

```diff
@@ -0,0 +1,14 @@
+'''
+문제 링크:https://www.acmicpc.net/problem/1546
+카테고리:수학, 사칙연산
+문제해설: 조작된 성적 평균 구하기
+'''
+a=int(input())
+b=list(map(int,input().split()))
+
+result=0
+
+for i in b:
+  result+=i/max(b)*100
+
```

Comment:
```suggestion

max_b = max(b)
for i in b:
  result+=i/max_bb*100

```

max_b와 같이 O(n)인 연산을 for문에서 항상 다시 계산하고 있으니 for 문 전에 변수에 담아두면 좋을 것 같네요